### PR TITLE
feat(vector): Use relative URL as basemaps stylejson sources.

### PR DESCRIPTION
**Pull request by Wentao-Kuang**

### CODE REVIEW

**Comment by Wentao-Kuang** [comment]
Apply this change to dev after merge https://github.com/linz/basemaps/pull/2032.

**Comment by Wentao-Kuang** [comment]
@blacha I have uploaded this into dev, which fixed the broken url. We can merge this after the prod release.

**Review by blacha — APPROVED** [review]

**Pull request merged**
